### PR TITLE
💚 Fixed GitHub action trigger in rolling build when CRM change happen

### PR DESCRIPTION
**Pull request by jernejk**

### CODE REVIEW

File: .github/workflows/rolling_build.yml [hunk 1]
```diff
@@ -45,11 +46,17 @@ jobs:
     # Trigger the other workflow if the CRM data was updated
     - name: Build and deploy People
       if: steps.determine-deployment-trigger.outputs.should_trigger_deploy == 'TRUE'
-      uses: ./.github/workflows/main-trigger-people-deploy.yml
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: main-trigger-people-deploy.yml
+        repo: SSWConsulting/SSW.People.Profiles
+        ref: ${{ steps.determine-deployment-trigger.outputs.branch_name }}
+        token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (wicksipedia) [inline .github/workflows/rolling_build.yml:52]: this should be using the app token instead of a PAT

Comment (wicksipedia) [inline .github/workflows/rolling_build.yml:52]: can we pass something so that it's easy to identify that it was triggered by a change in CRM skills?

Comment (jernejk) [reply .github/workflows/rolling_build.yml:52]: As per our conversation, I've added https://github.com/SSWConsulting/SSW.People/issues/655

**Review by wicksipedia — APPROVED** [review]
🤞 I expect that this will have an issue triggering the workflow but happy to be mistaken

**Comment by jernejk** [comment]
> 🤞 I expect that this will have an issue triggering the workflow but happy to be mistaken

@wicksipedia, it worked before I created the PR. :)
I couldn't do that with the previous PR because I didn't have the option to trigger this build from a different branch, and now I can.

**Pull request merged**